### PR TITLE
update ingress status after loaded

### PR DESCRIPTION
**Pull request by witskeeper**
#1910

### CODE REVIEW

**Comment by witskeeper** [comment]
Any suggestions ？

**Review by MihaZupan — APPROVED** [review]
Mainly just style nits, otherwise LGTM, thanks

File: src/Kubernetes.Controller/Client/IIngressResourceStatusUpdater.cs [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+using System;
+using System.Threading;
+using System.Threading.Tasks;
+
+namespace Yarp.Kubernetes.Controller.Client;
+
+public interface IIngressResourceStatusUpdater
+{
+    /// <summary>
+    /// <see cref="IIngressResourceStatusUpdater"/>update the cached ingress status
```
Comment (MihaZupan) [inline src/Kubernetes.Controller/Client/IIngressResourceStatusUpdater.cs:16]:
```suggestion
    /// Updates the status of cached ingresses.
```

**Comment by witskeeper** [comment]
so, can we merge this pr now ?

**Comment by karelz** [comment]
@witskeeper Miha should be able to get back to it on Fri / early next week. Thanks for your patience.

**Comment by MihaZupan** [comment]
Sorry about the extra wait, I was out for the week.

**Comment by amitay-elementor** [comment]
The changes here seem to update all ingresses including ones managed by a different ingress classes.
Checking `IsYarpIngress` in the ingress loop should fix the issue

**Pull request merged**